app.js

```diff
@@ -111,63 +48,43 @@ var storeHours = [
   '7:00 PM',
 ];
 
-var firstAndPiketotalCookies = 0;
-var firstAndPikeListElement = document.getElementById('firstAndPikeList');
+var headerEl = document.createElement('thead');
+var headerRow = document.createElement('tr');
```

Comment:
from the assignment instructions: The header row and footer row are each created in their own stand-alone function